<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_db_status">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>ðŸ”´ Desconectado</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777212</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>6</number>
        </property>
        <property name="tabSpacing">
         <number>7</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>169</width>
           <height>339</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <attribute name="label">
          <string>Servidor</string>
         </attribute>
         <widget class="QPushButton" name="btn_db_connect">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>100</y>
            <width>151</width>
            <height>28</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Conectar base de datos</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_host">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>10</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Host</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Usuario</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>ContraseÃ±a</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_user">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>40</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_password">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>70</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_db_disconnect">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>151</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Desconectar</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>169</width>
           <height>339</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <attribute name="label">
          <string>Herramientas</string>
         </attribute>
         <widget class="QPushButton" name="btn_validate_full_day">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Validar todo</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>169</width>
           <height>339</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Queries</string>
         </attribute>
         <widget class="QPushButton" name="btn_show_gp_table">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>91</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>GP</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Ver tabla:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_show_mbd_table">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>91</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>MBD</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_show_drg_table">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>80</y>
            <width>91</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Drg</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>169</width>
           <height>339</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Matenimiento base de datos</string>
         </attribute>
         <widget class="QPushButton" name="btn_update_db_mbd">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>150</width>
            <height>28</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Actualizar MBDs</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEdit">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>110</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_update_db_gp">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>100</y>
            <width>150</width>
            <height>28</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Actualizar GPs</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>161</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Actualizar a partir del dÃ­a:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_update_db_drg">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>130</y>
            <width>150</width>
            <height>28</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Actualizar Drgs</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>169</width>
           <height>339</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Faltas</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="page_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>169</width>
           <height>339</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="page_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>169</width>
           <height>339</height>
          </rect>
         </property>
         <attribute name="label">
          <string>ConfiguraciÃ³n</string>
         </attribute>
         <widget class="QPushButton" name="btn_toggle_console">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>110</y>
            <width>171</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Esconder/mostrar consola</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_move_table">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Mover tabla al mover dÃ­a</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Ver calendario</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>161</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Solo ver GP vÃ¡lidos</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_edit_table">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>131</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Editar tabla</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTextBrowser" name="log_window">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="line_query"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_run_query">
          <property name="text">
           <string>Ejecutar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_move_back">
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_current_date">
          <property name="text">
           <string>2/02/2022</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_move_forward">
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>MBD:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Drg:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Comentarios:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/sergio.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_sergio">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/aitor.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_aitor">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/joaquin.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_joaquin">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/paula.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_paula">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/miranda.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_miranda">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/diego.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_diego">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/nerea.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_nerea">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/laura.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_laura">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/anton.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_anton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>assets/aina.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gp_aina">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>assets/gp_icon.ico</normaloff>assets/gp_icon.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_show_current_mbd">
        <property name="text">
         <string>Ver MBD</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
